homeassistant/components/generic/config_flow.py

```diff
@@ -170,26 +162,40 @@ async def async_test_still(
             response = await async_client.get(url, auth=auth, timeout=GET_IMAGE_TIMEOUT)
             response.raise_for_status()
             image = response.content
+            content_type = response.headers["content-type"]
     except (
         TimeoutError,
         RequestError,
         HTTPStatusError,
         TimeoutException,
+        KeyError,
     ) as err:
         _LOGGER.error("Error getting camera image from %s: %s", url, type(err).__name__)
```

Comment:
Might be worth adding `, exc_info=True` here - getting just "Error getting camera image from ...: KeyError" is not much help with debugging.

Reply:
Agreed, done afae319.